### DISCUSSION TITLE
Programmation d'une visio courant septembre ?

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#19] [General] [answered]
Hello tout le monde,
J'espère que vous allez bien.

Est-il possible de se réunir pour discuter directement de différents points ? Des tâches à faire ? ...

Personnelement, je me suis permise de faire une copie du repos pour bosser et je dois avouer qu'il y a quelques changements que j'aimerai vous soumettre.
2. Que la préface se retrouve en page d'acceuil (j'aiessayé de retravailler dessus)
3. Ajout d'une fonction wikilinks déjà développé avec le lexique (infobulle au survol)
4. Modification du nommage du cycle de vie et faire qu'une bonne patrique soit associée à un seul + filtre (j'arrivais plus à m'y retrouver c'est pourquoi j'ai fait les modifications).

Modification de BP :
- Est ce que Limiter le nombre d'extension, n'est pas limiter le nombre de fonctionnalités au final ?
- Etendre la limitation de la vidéo à la limitation de média (image, carte etc.) ou faire de nouvelles BP
- Réunir la limitation de poids du css/jss + minifier

Ajout de BP :
- Simplifier la navigation
- Je ne suis pas spécialiste de wordpress mais il me semble qu'il y a des plugins de save et backup. Est ce utile sachant que l'hébergeur fait des save tous les jours ? Quoique quand OVH a pris feu ?

Je pense que c'est un peu comme vous, depuis juin je suis difficilement disponible.
Bien à vous

**Comment by dNicolle** [comment, marked as answer]
@florinesueur 
Oui, on peut prévoir une réunion avant la fin du mois, je fais un mail dans la foulée.

Pour ma part, je suis OK pour :
 2. Que la préface se retrouve en page d'acceuil (j'aiessayé de retravailler dessus) 
     > Tu veux dire, faire un lien depuis https://github.com/cnumr/fiches-wordpress-greenit/tree/main/best-practices-WordPress vers la page de Préface, ou mettre la préface à la place de cela ?
  3. Ajout d'une fonction wikilinks déjà développé avec le lexique (infobulle au survol)
     > Bonne idée oui, il faut alimenter le glossaire donc :)

Modification de BP :
 - oui limiter les extentions, c'est limiter les fonctionnalités mais pas que, parfois, plusierus extensions d'un même site vont faire pour partie la même chose, ou vont restez installée sans être utilisées
 - faire une nouvelle Bp pour limiter les médias en général me semble plus pertinent puisque les infos liées à la vidéo sont assez spécifique, les éléments à transmettre ne sont pas les même selon moi.
  - Les fiches sur le CSS et JS sont en cours d'écriture par Yann K., j'attendrais bien de voir son contenu avant de savoir si elles doivent être réunies
  
Ajout de BP
 - yes, tu te sens de les écrire ?

**Reply by hrenaud** [reply]
@florinesueur tu devrais faire un FORK du repo pour pouvoir faire tes modifs dans ton coin et faire une MR pour la réintégrer dans le repo global.

> Si tu changes beaucoup de choses, il va surement falloir que je redéveloppe certaines choses pour adapter. Il faudra qu'on en discute à ce moment-là.

Concernant les plugins de backup, ajouté à un site, ils permettent, si l'utilisateur à plusieurs sites, de ne récupérer que celui-ci plutôt que d'écraser tous les sites avec le backup de l'hébergeur qui ne propose pas toujours de granularité.
